CurrentCard/docs/README.md

```diff
@@ -25,13 +25,13 @@ The high level block diagram of the current sensor card is shown below:
 <img src="images/current-sensor-blockdiagram.svg" width="500%" />
 
 ### Current Sensor
-LEM LA 55-P current sensor is selected for this design, as it is the only sensor available from LEM with an open aperture and PC pins that can measure +/-55A. 
+LEM LA 55-P and LA 100-P current sensors are selected for this design, as it is the only sensor available from LEM with an open aperture and PC pins that can measure +/-55A and +/-150A, respectively. 
```

Comment:
@noguchi-takahiro this sentence as is isn't quite grammatically correct.
Can you improve it to say that Either LA-55P or LA 100-P can be used with this design based on the sensing range desired (and provide the sensing range as you are currently doing).

Also please include a note saying that LA-100P has a variant SP/13 which has a different gain that users must be careful about when ordering.